core/src/main/java/com/amplifyframework/core/model/AuthRule.java

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.core.model;
+
+import java.util.Arrays;
+import java.util.Objects;
```

Comment:
`Objects` requires minSdk of 19, doesn't it? We are using `ObjectsCompat` for this reason in much of the code base. Nevermind that there are currently issues with APIs 16-19... might as well not put gas on the bonfire. 😄 🚀

Reply:
Fixed!   This was for the IDE-generated implementation of `equals()`.  I'll make note to replace with `ObjectsCompat` next time.